README.md

```diff
@@ -131,6 +131,37 @@ git push -u origin <your_branch_name>
 
 ---
 
+### Frequently Asked Questions(FAQs) for GSSoC'20:
+
+#### 1. How to start as a beginner so as to contribute to the project?
+There are many beginner level issues to begin with, which include documentation, development and design. Also, feel free to make issues for features as well as for reporting bugs.Participants can claim an issue by commenting that they are interested in it. We are also open to new ideas and suggestions.
+
+
+#### 2. Is this a hardware/IoT project?
+Yes, it is a hardware/IoT + software project. There would be maximum tasks related to frontend, backend, design and documentation of the website and later, it would be integarted with the real-time data from the sensors.
+
+#### 3.How to communicate with us?
```

Comment:
@PragatiVerma18  Missing whitespace between numbering and the first character of the question.